defenseunicorns-labs/lula1
# feat(deps): add support for OSCAL 1.1.3 as default

## Description

Switches to the new go-oscal OSCAL 1.1.3 types

quick find-and-replace for `oscalTypes_1_1_2` switching to `oscalTypes` to prevent necessary modification in the future. 

We have future issues to address the use of non-latest OSCAL versions and evaluation of what that means for go-oscal and Lula. 

Tested the upgrade/lint workflow to ensure they operated accordingly. 

## How to test these changes manually

- Confirm build without issues
- Search repository for "github.com/defenseunicorns/go-oscal/src/types/oscal-1-1-2"
  - No matches should exist
- Search repository for "oscalTypes_1_1_2"
  - No matches should exist
- Tests passing

## Related Issue

Fixes #834


## Type of change


## Checklist before merging

- [x] [Schema Updates](https://github.com/defenseunicorns/lula/blob/main/docs/community-and-contribution/schema-updates.md) applied
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/lula/blob/main/CONTRIBUTING.md) followed
